### ISSUE TITLE
Sort out team id that we can both use

### ISSUE THREAD

**Issue by thomashorrobin** [dylankbuckley/monzo-menubar-macos#2]
I'm not entirely sure how this works in XCode but I think we need a team cert that we can both use. Currently I just change the cert to my personal local one everytime I do a pull but it's kinda annoying to be constantly changing it

**Comment by dylankbuckley** [comment]
Interestingly when I pulled from your branch, xcode didn't ask me to do anything. It just worked. I think it's just the case of creating a team cert and then sharing it to eachothers keychain.

**Comment by thomashorrobin** [comment]
Yeah it worked for you because I set the team id back to yours before I pushed. I'll have to read up about how to do this properly lol
